src/transformers/tsx.ts

```diff
@@ -15,26 +15,45 @@ export default function transformTsx(code: string, node: Node, cssRoot: Root): T
       s.overwrite(match.index, match.index + match[1].length + 2,`= \`${utilityClasses} \${${match[1]}}\``)
     }
     return {
-      code: s.toString().replace(/\\:/g, '\\\\:')
+      code: s.toString().replace(/\\/g, '\\\\')
     }
   }
 }
 
 function _buildTailwindClassList(node: Node, cssRoot: Root): string | undefined {
   const classes = _parseClasses(node)
   if (classes.length) {
-    return cssRoot?.nodes?.filter(isRule).reduce((acc: any, rule: Rule) => {
-      const match = rule.selector.replace(/\\/, '').match(/([a-zA-Z0-9-]+$|[a-zA-Z0-9-]+:[a-zA-Z0-9-]+)/)
-      if (match && classes.includes(match![0])) {
-        return rule.toString().replace(/\s+/gm, ' ') + ' \\n' + acc
+    const nodes = cssRoot?.nodes?.filter(isRule).filter(rule => {
+      const match = rule.selector.replace(/\\/g, '').match(/([a-zA-Z0-9-]+$|[a-zA-Z0-9-]+:[a-zA-Z0-9-]+)/)
+      if (match && classes.includes(match[0])) {
+        return rule
       }
-      return acc
-    }, '')
+    });
+
+    const atRules = cssRoot?.nodes?.filter(isAtRule).map(atRule => {
```

Comment:
This was the crux of the issue, the responsive classes are hosted inside at rule (e.g. `@media min-size(123px)`) so they weren't correctly being added to the final CSS.

Reply:
Maybe remove the console.log at line 26? 😃